src/Settings/SettingsInfo.hpp

```diff
@@ -65,7 +65,7 @@ class SettingsInfo
   private:
     static QList<SettingInfo> settings;
 
-    friend class SettingsUpdater;
+    friend struct SettingsUpdater;
```

Comment:
`SettingsUpdater` is defined as `struct` so friend declaration should be `struct` not `class`